app/models/dialog_field_drop_down_list.rb

```diff
@@ -1,4 +1,6 @@
 class DialogFieldDropDownList < DialogFieldSortedItem
+  AUTOMATE_VALUE_FIELDS = %w(required read_only visible).freeze
```

Comment:
I don't think this constant is necessary, since we're inheriting from DialogFieldSortedItem which should already have a constant that contains all of the proper automate value fields